### PR TITLE
osd: add perf counter to record osd slow ops

### PR DESCRIPTION
**Pull request by Rethan**
Use a seperate slow op counter to record total slow ops.

Signed-off-by: haoyixing <haoyixing@kuaishou.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by Rethan** [comment]
jenkins retest this please

**Comment by Rethan** [comment]
jenkins retest this please

**Comment by Rethan** [comment]
jenkins retest this please

**Comment by Rethan** [comment]
jenkins retest this please

**Comment by Rethan** [comment]
jenkins retest this please

**Comment by Rethan** [comment]
Rebased on main.

File: src/osd/PrimaryLogPG.cc [hunk 1]
```diff
@@ -4378,6 +4378,9 @@ void PrimaryLogPG::log_op_stats(const OpRequest& op,
   const utime_t process_latency = now - op.get_dequeued_time();
 
   osd->logger->inc(l_osd_op);
+  if (latency >= g_conf()->osd_op_complaint_time) {
```
Comment (ifed01) [inline src/osd/PrimaryLogPG.cc:4381]: most of places I've checked use op.get_initiated() as a starting point for complaint time. And you use m->get_recv_stamp() call above to set latency var instead.  For the sake of uniformity may be use get_initiated() here as well?

Comment (Rethan) [reply src/osd/PrimaryLogPG.cc:4381]: Updated.

**Review by ifed01 — APPROVED** [review]

**Pull request closed without merging**